App.News/Controllers/NewsController.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Mvc;
+using App.News.Models;
+using App.News.Interfaces;
+
+namespace App.News.Controllers
+{
+    [Route("api/news")]
+    [ApiController]
+    public class NewsController:Controller
+    {
+        INewsManager newsManager;
+
+        public NewsController(INewsManager newsManager)
+        {
+            this.newsManager = newsManager;
+        }
+        [HttpPost]
+        public ActionResult AddComment([FromBody]CommentDTO commentDTO)
+        {
+            var actionResult = newsManager.AddComment(commentDTO);
+            if (actionResult != "ok")
+            {
+                return new BadRequestResult();
+            }
+            return new OkResult();
+        }
+        [Route("api/news/allNews")]
+        [HttpGet]
+        public ActionResult<IEnumerable<NewsDTO>> GetNews() {
```

Comment:
1. You no need downcast before return result.
2. You no need create OkObjectResult class instance. You can write something like this - 
**return Ok(newsManager.GetAllNews());**